nova/compute/api.py

```diff
@@ -2543,14 +2543,6 @@ def _filter(instance):
             objects=list(filter(filter_method,
                            build_req_instances.objects +
                            insts.objects)))
```

Comment:
I'd really prefer this to be `[:orig_limit]` instead of changing the unique filter to do something that doesn't match its name.  Works with `None`, too:
```python
>>> a = [1, 2, 3, 4]
>>> a[:None]
[1, 2, 3, 4]
```